src/xcode/ENA/ENA/Source/Services/CCLService/GetWalletInfoInput.swift

```diff
@@ -0,0 +1,45 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import AnyCodable
+import CertLogic
+
+struct SystemTime: Codable {
+	
+	let timestamp: Int
+	let localDate: String
+	let localDateTime: String
+	let localDateTimeMidnight: String
+	let utcDate: String
+	let utcDateTime: String
+	let utcDateTimeMidnight: String
+}
+
+enum GetWalletInfoInput {
+	
+	static func make(
+		with date: Date = Date(),
+		language: String,
+		certificates: [DCCWalletCertificate],
+		boosterNotificationRules: [Rule]
+	) -> [String: AnyDecodable] {
+		let systemTime = SystemTime(
+			timestamp: Int(Date().timeIntervalSince1970),
+			localDate: DateFormatter.localDateFormatter.string(from: date),
+			localDateTime: DateFormatter.localDateTimeFormatter.string(from: date),
+			localDateTimeMidnight: DateFormatter.localDateMidnightTimeFormatter.string(from: date),
+			utcDate: DateFormatter.utcDateFormatter.string(from: date),
+			utcDateTime: DateFormatter.utcDateTimeFormatter.string(from: date),
+			utcDateTimeMidnight: DateFormatter.utcDateTimeMidnightFormatter.string(from: date)
+		)
```

Comment:
Looks to me like it could make sense to have an initializer for `SystemTime` with a date. I think we will need to reuse this for the other function calls.

Reply:
Done